src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/AlleleSubsettingUtils.java

```diff
@@ -306,6 +309,10 @@ else if ( clazz.equals(int[].class) ) {
         }
 
         final double[] likelihoodSums = calculateLikelihoodSums(vc, defaultPloidy, allHomRefData);
+        if (MathUtils.sum(likelihoodSums) == 0.0 && !allHomRefData) {
+            throw new IllegalStateException("No likelihood sum exceeded zero -- method was called for variant data " +
+                    "with no variant information.");
```

Comment:
Are users likely to encounter this error in practice? If yes, maybe add a bit more contextual info to make it less cryptic (such as the fact that we're doing allele subsetting and calculating the most likely alleles)